drivers/SmartThings/zigbee-button/src/test/test_shinasystem_button.lua

```diff
@@ -217,16 +218,20 @@ test.register_coroutine_test(
           mock_device.id,
           PowerConfiguration.attributes.BatteryVoltage:configure_reporting(mock_device, 30, 21600, 1)
     })
-    test.socket.zigbee:__expect_send({
-      mock_device.id,
-      PowerConfiguration.attributes.BatteryVoltage:read(mock_device)
-    })
```

Comment:
I also wonder where read command is invoked like in other TC(https://github.com/SmartThingsCommunity/SmartThingsEdgeDriversBeta/blob/master/drivers/archive/zigbee-button/src/test/test_samjin_button.lua#L45-L53)

If I don't add `local do_configure = function(self, device)` of this PR, this read command is called internally.
But I found that it wasn't called even if I added device:configure() internally.
So, I added 'read' before add_hub_to_zigbee_group, explicitly.
Because for this sensor, the read command is necessary during the configuring to get the battery status.